From 3fb89274c9d618c562a90f26a47f2c5cb315d7b1 Mon Sep 17 00:00:00 2001
From: Wu Liangqing <wlq@rock-chips.com>
Date: Fri, 30 Oct 2020 11:23:00 +0800
Subject: [PATCH] net: rockchip_wlan: fix clang compile error

drivers/net/wireless/rockchip_wlan/*/core/rtw_debug.c:45:44:
error: expansion of date or time macro is not reproducible [-Werror,-Wdate-time]
            RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);

Signed-off-by: Wu Liangqing <wlq@rock-chips.com>
Change-Id: I506498ea3013ee3458dc1f8d2a88d9aac76c149e
---
 drivers/net/wireless/rockchip_wlan/rtl8189es/core/rtw_debug.c | 2 +-
 drivers/net/wireless/rockchip_wlan/rtl8189fs/core/rtw_debug.c | 2 +-
 drivers/net/wireless/rockchip_wlan/rtl8723bs/core/rtw_debug.c | 2 +-
 drivers/net/wireless/rockchip_wlan/rtl8723cs/core/rtw_debug.c | 2 +-
 drivers/net/wireless/rockchip_wlan/rtl8723ds/core/rtw_debug.c | 2 +-
 drivers/net/wireless/rockchip_wlan/rtl8822be/core/rtw_debug.c | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/net/wireless/rockchip_wlan/rtl8189es/core/rtw_debug.c b/drivers/net/wireless/rockchip_wlan/rtl8189es/core/rtw_debug.c
index ac165df1aedb..01c0317e1bea 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8189es/core/rtw_debug.c
+++ b/drivers/net/wireless/rockchip_wlan/rtl8189es/core/rtw_debug.c
@@ -68,7 +68,7 @@ u32 GlobalDebugLevel = _drv_err_;
 void dump_drv_version(void *sel)
 {
 	DBG_871X_SEL_NL(sel, "%s %s\n", DRV_NAME, DRIVERVERSION);
-	DBG_871X_SEL_NL(sel, "build time: %s %s\n", __DATE__, __TIME__);
+	//DBG_871X_SEL_NL(sel, "build time: %s %s\n", __DATE__, __TIME__);
 }
 
 void dump_drv_cfg(void *sel)
diff --git a/drivers/net/wireless/rockchip_wlan/rtl8189fs/core/rtw_debug.c b/drivers/net/wireless/rockchip_wlan/rtl8189fs/core/rtw_debug.c
index 031ed0f02dc5..a3252da84313 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8189fs/core/rtw_debug.c
+++ b/drivers/net/wireless/rockchip_wlan/rtl8189fs/core/rtw_debug.c
@@ -43,7 +43,7 @@ const char *rtw_log_level_str[] = {
 void dump_drv_version(void *sel)
 {
 	RTW_PRINT_SEL(sel, "%s %s\n", DRV_NAME, DRIVERVERSION);
-	RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);
+	//RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);
 }
 
 void dump_drv_cfg(void *sel)
diff --git a/drivers/net/wireless/rockchip_wlan/rtl8723bs/core/rtw_debug.c b/drivers/net/wireless/rockchip_wlan/rtl8723bs/core/rtw_debug.c
index 03202f80d729..056d6740f366 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8723bs/core/rtw_debug.c
+++ b/drivers/net/wireless/rockchip_wlan/rtl8723bs/core/rtw_debug.c
@@ -42,7 +42,7 @@ const char *rtw_log_level_str[] = {
 void dump_drv_version(void *sel)
 {
 	RTW_PRINT_SEL(sel, "%s %s\n", DRV_NAME, DRIVERVERSION);
-	RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);
+	//RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);
 }
 
 void dump_drv_cfg(void *sel)
diff --git a/drivers/net/wireless/rockchip_wlan/rtl8723cs/core/rtw_debug.c b/drivers/net/wireless/rockchip_wlan/rtl8723cs/core/rtw_debug.c
index d1a41e1908dd..c6ae9b1a2e3d 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8723cs/core/rtw_debug.c
+++ b/drivers/net/wireless/rockchip_wlan/rtl8723cs/core/rtw_debug.c
@@ -42,7 +42,7 @@ const char *rtw_log_level_str[] = {
 void dump_drv_version(void *sel)
 {
 	RTW_PRINT_SEL(sel, "%s %s\n", DRV_NAME, DRIVERVERSION);
-	RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);
+	//RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);
 }
 
 #ifdef CONFIG_PROC_DEBUG
diff --git a/drivers/net/wireless/rockchip_wlan/rtl8723ds/core/rtw_debug.c b/drivers/net/wireless/rockchip_wlan/rtl8723ds/core/rtw_debug.c
index 0110d40436b0..20a5b02b649e 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8723ds/core/rtw_debug.c
+++ b/drivers/net/wireless/rockchip_wlan/rtl8723ds/core/rtw_debug.c
@@ -43,7 +43,7 @@ const char *rtw_log_level_str[] = {
 void dump_drv_version(void *sel)
 {
 	RTW_PRINT_SEL(sel, "%s %s\n", DRV_NAME, DRIVERVERSION);
-	RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);
+	//RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);
 }
 
 void dump_drv_cfg(void *sel)
diff --git a/drivers/net/wireless/rockchip_wlan/rtl8822be/core/rtw_debug.c b/drivers/net/wireless/rockchip_wlan/rtl8822be/core/rtw_debug.c
index 8c978176065b..60e4eae9ee37 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8822be/core/rtw_debug.c
+++ b/drivers/net/wireless/rockchip_wlan/rtl8822be/core/rtw_debug.c
@@ -47,7 +47,7 @@ const char *rtw_log_level_str[] = {
 void dump_drv_version(void *sel)
 {
 	RTW_PRINT_SEL(sel, "%s %s\n", DRV_NAME, DRIVERVERSION);
-	RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);
+//	RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);
 }
 
 void dump_drv_cfg(void *sel)
-- 
2.35.3

